include/seccomp.h.in

```diff
@@ -35,6 +37,24 @@
 extern "C" {
 #endif
 
+/*! \mainpage libseccomp
+ *
+ * \section intro_sec Introduction
+ *
+ * @c The libseccomp library provides an easy to use, platform independent,
+ * interface to the Linux Kernel's syscall filtering mechanism. The libseccomp
+ * API is designed to abstract away the underlying BPF based syscall filter
+ * language and present a more conventional function-call based filtering 
+ * interface that should be familiar to, and easily adopted by, application
+ * developers.
+ *
+ * View the entire
+ * <a href="https://github.com/seccomp/libseccomp/blob/main/README.md">README</a>
+ * for more information
+ *
+ * Libcgroup's C APIs are outlined in the @ref APIs "APIs" section.
```

Comment:
I'm pretty sure you meant "libseccomp" here.

At least we know where you got the boilerplate ;)

Reply:
Hmmmm.... :)

Reply:
I hate to say it, but I actually typed that by hand.  Can't even blame copy/paste

Reply:
Fixed